src/java/org/apache/cassandra/config/Config.java

```diff
@@ -193,6 +193,8 @@
     public volatile boolean native_transport_allow_older_protocols = true;
     public volatile long native_transport_max_concurrent_requests_in_bytes_per_ip = -1L;
     public volatile long native_transport_max_concurrent_requests_in_bytes = -1L;
+    public volatile boolean native_transport_rate_limiting_enabled = false;
+    public volatile double native_transport_requests_per_second = Double.MAX_VALUE;
```

Comment:
Note: I've not yet added the new parameters to `cassandra.yaml`, but `native_transport_max_concurrent_requests_in_bytes` and `native_transport_max_concurrent_requests_in_bytes_per_ip` also don't exist there yet. Perhaps this is due to these features being somewhat experimental, but I'm not sure where the threshold is for adding them.